src/views/Blog.vue

```diff
@@ -1,9 +1,29 @@
 <template>
-    <div class="flex justify-center font-bold text-3xl mt-20">
-        All the blogs will show here
+    <div v-for="item in data" :key="item.id" class="flex flex-col text-center justify-center mt-20">
+        <h3 class="text-2xl font-bold">{{ item.title }}</h3> 
+        <p>{{ item.description }}</p>
+        <div class="flex flex-row gap-4 mx-auto w-1/12 mt-4">
+            <button class=" py-2 px-4 rounded hover:bg-red-500 hover:text-white hover:border-none border-gray-500">Delete</button>
+            <button class=" py-2 px-4 hover:bg-green-500 hover:text-white hover:border-none rounded border-gray-500">Update</button>
+        </div>
     </div>    
 </template>
 
 <script setup>
+import { onMounted, ref } from 'vue'
+import Blogs from '../composables/GetBlog'
+import { RouterLink } from 'vue-router';
 
+const { getBlog  } = Blogs()
```

Comment:
We could have a file containing different functions exported and could import directly here. without calling `Blogs()`